_data/events_gdcr2018/Poznan.json

```diff
@@ -0,0 +1,17 @@
+{
+  "title": "GDCR 2018 Poznań",
+  "url": "",
+  "moderators": [
+    "https://www.meetup.com/Poznan-Java-User-Group/events/255433496/"
```

Comment:
This link should be the value of the `url` key above.
In `moderators` you should add the facilitators twitter handles or just their full names (if they do not use twitter)

Reply:
Sure, sorry for the mistake!